radixconfig.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: radix.equinor.com/v1
+kind: RadixApplication
+metadata:
+  name: flotilla
+spec:
+  build:
+    secrets:
+      - TLS_SERVER_KEY
+  environments:
+    - name: dev
+      build:
+        from: main
+    - name: qa
+    - name: prod
+  components:
+    - name: frontend
+      src: frontend
+      dockerfileName: Dockerfile_Radix
```

Comment:
Should it be called DockerfileRadix without the _?

Reply:
Maybe, i was thinking that since a normal dockerfile is simply called `Dockerfile`, it would make it clearer what this was if I divided it more clearly like this

Reply:
Could we then just call it Radix as Dockerfile is implied by dockerfileName?

Reply:
Just realized that it points to the file Dockerfile_Radix... The way I have seen it earlier have been to call it Dockerfile.radix?

Reply:
Will change!

Reply:
fixed